Makefile

```diff
@@ -35,8 +35,12 @@ CONFIG_VARS = curl -fsL https://ft-next-config-vars.herokuapp.com/$1/$(call APP_
 
 # clean
 clea%:
-# HACK: Can't use -e option here because it's not supported by our Jenkins
-	@git clean -fxd
+	# clean just gitignored files if not on CircleCI
+	@if [ -z $(CIRCLECI) ]; then git clean -Xfd; else git clean -xfd; fi
+	@$(DONE)
+
+cleanx:
```

Comment:
`make new`
`make tidy`
`make fresh`
`make i-give-up-take-me-back-to-how-it-was-before`